src/components/table-header.js

```diff
@@ -52,6 +62,21 @@ const TableHeader = ({
     const currentTime = dateFormat(timestamp, 'HH:MM');
     const outlined = taskStatus === 'RUNNING' ? 'outlined' : 'filled';
     const tableHeaderName = (processName || '') + ' Supervisor';
+    const [manualExportEnabled, setManualExportEnabled] = useState([]);
+
+    useEffect(() => {
+        async function getManualExportEnabled() {
+            let filter = await fetch('process-metadata.json')
+                .then((res) => {
+                    return res.json();
+                })
+                .then((res) => {
```

Comment:
You can improve the code by making it more concise and readable. Here's a refactored version using async/await directly instead of succession of .then() :

```
 useEffect(() => {
    async function getManualExportEnabled() {
        try {
            const response = await fetch('process-metadata.json');
            if (!response.ok) {
                throw new Error(
                    `Fetch request failed with status: ${response.status}`
                );
            }
            const data = await response.json();
            const manualExportEnabled = data.manualExportEnabled || false;
            setManualExportEnabled(manualExportEnabled);
        } catch (error) {
            console.error('An error has occurred:', error);
            // Handle the error as needed
        }
    }
    getManualExportEnabled();
}, []);
```

But I leave you the choice to choose the writing that seems clearest to you, 
In any case, both ways are not ideal, we could have written the code better and managed errors better but that requires a lot of refactoring